product_docs/docs/transporter/ba_preview/getting_started/create_database.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Creating a database cluster"
+---
+
+Before you migrate your database to EDB Postgres® AI, create an empty database in a database cluster. 
+You will configure this database to be the target of your migration when using EDB Transporter. 
+
+1.  Access the [EDB Postgres AI Console](https://portal.biganimal.com) and log in with your EDB Postgres AI Database Cloud Service credentials.
+
+1.  Select the project where you want to create the database cluster. 
+
+    See [Creating a project](/biganimal/latest/administering_cluster/projects/#creating-a-project) if you want to create one.
+
+1.  Within your project, select **Create New** and **Database Cluser** to create an instance that will serve as target for the EDB Transporter migration.
+
+    See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/) for detailed instructions on how to create a single-node or a primary/standby high availability cluster. 
+
+    See [Creating a distributed high-availability cluster](/biganimal/latest/getting_started/creating_a_cluster/creating_a_dha_cluster/) for detailed instructions on how to create a distributed high availibility cluster.
```

Comment:
My colleague @djw-m was wondering if Transporter has been tested on PGD clusters? Otherwise we should be explicit that this type of cluster is not supported.

Reply:
yes, it's working with PGD